### DISCUSSION TITLE
The inversion effect of three-dimensional flow field is not good

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#698] [General]
What parameters can be adjusted to improve the poor inversion effect of three-dimensional flow field examples

**Comment by friedenhe** [comment]
We discussed this in the [PoF paper](https://www.researchgate.net/publication/379889817_Field_Inversion_Machine_Learning_Augmented_Turbulence_Modeling_for_Time-Accurate_Unsteady_Flow).
